### PR TITLE
100% RU locale

### PR DESCRIPTION
**Pull request by krotesk**

### CODE REVIEW

File: packages/tools/locales/ru_RU.po [hunk 1]
```diff
@@ -5091,377 +5086,3 @@ msgstr "Приблизить"
...
-
-#~ msgid "Separate each tag by a comma."
-#~ msgstr "Разделяйте каждую метку запятой."
-
-#~ msgid ""
-#~ "The path to synchronise with when file system synchronisation is enabled. "
-#~ "See `sync.target`."
-#~ msgstr ""
-#~ "Путь для синхронизации, когда включена синхронизация файловой системы. "
-#~ "См. `sync.target`."
```
Comment (tessus) [inline packages/tools/locales/ru_RU.po:5467]: Please don't delete those. We keep old translations in the .po files.

Comment (krotesk) [reply packages/tools/locales/ru_RU.po:5467]: Okay. I just downloaded the .pot file and made a "rough translation" of the already translated .po

Comment (tessus) [reply packages/tools/locales/ru_RU.po:5467]: What? I am very confused. Everything in the PR is great. Just don't delete the commented out strings. You only use the .pot file, if you start a new translation.

P.S.: What you do is that you download the .po file and load it into poedit and add the missing translations. Done.

Comment (tessus) [reply packages/tools/locales/ru_RU.po:5467]: Are you still working on this?

**Comment by tessus** [comment]
btw, there also seems something off. the first commit is several months old.

Maybe you should read: https://discourse.joplinapp.org/t/how-to-create-a-pull-request-pr/15052?u=tessus

**Comment by tessus** [comment]
I'll be closing this PR, if there's no update by Wednesday.

**Pull request closed without merging**